charts/sessionspaces/charts/sessionspaces/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: sessionspaces-clusterrolebinding
```

Comment:
We don't need to affix the kind here. Would probably be best just to use the `sessionspaces.fullname`